src/createDispatcher.js

```diff
@@ -1,8 +1,12 @@
 import composeMiddleware from './utils/composeMiddleware';
 
+const INIT_ACTION = {
+  type: '@@INIT'
```

Comment:
I was actually kinda torn on this one. :-)
Action types are all caps by convention.

I'd rather keep `@@` as the marker of “special stuff” and keep using all caps for action types and camel for state keys:

```js
export const StateKeys = {
  LOG: '@@log'
};

export const ActionTypes = {
  JUMP_TO_STATE: '@@JUMP_TO_STATE'
};
```